src/get-started/java-binding.md

```diff
@@ -587,15 +604,13 @@ The TestKit also allows creating blocks that contain all current [in-pool][in-po
 transactions:
 
 ```java
-try (TestKit testKit = TestKit.forService(MyServiceModule.class)) {
-  // Put the transaction into the TestKit transaction pool
-  MyService service = testKit.getService(MyService.SERVICE_ID, MyService.class);
+// Instantiate a service that submits a transaction in it's `afterCommit` method
+try (TestKit testKit = TestKit.forService(ARTIFACT_ID, ARTIFACT_FILENAME,
+        SERVICE_NAME, SERVICE_ID, artifactsDirectory)) {
+  // Create a block so that `afterCommit` method is invoked
+  testKit.createBlock();
 
-  TransactionMessage message = constructTransactionMessage();
-  RawTransaction rawTransaction = RawTransaction.fromMessage(message);
-  service.getNode().submitTransaction(rawTransaction);
-
-  // This block will contain the transaction submitted above
+  // This block will contain the transaction submitted in `afterCommit` method
```

Comment:
```suggestion
  // The block will contain the transaction submitted in the `afterCommit` method
```

Reply:
Why `this` is bad though? It refers to the `Block block` below.